kubernetes/eventhandler.go

```diff
@@ -229,3 +264,9 @@ func (*nodePodUpdater) OnAdd(interface{}) {}
 // OnDelete handles delete events on namespaces. Nothing to do, if pods are deleted from this
 // namespace they will generate their own delete events.
 func (*nodePodUpdater) OnDelete(interface{}) {}
+
+// isEqualMetadata receives labels or annotations maps and checks their equality. Returns True if equal, False if there is a diffrence
```

Comment:
actually this function just compares two maps of <string,string>. Calling isEqualMetadata sounds like it's doing something smarter with the metadata. Also mentioning the labels and annotations in the function docs is wrong, since the function has no knowledge of the parameters being passed.

This function has now lost its reason to exists. Just inlining into the calling code it makes a lot more sense.

Reply:
I agree, I removed it